backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/lastbusinessday.sql

```diff
@@ -0,0 +1,68 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.migration-sql
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+DROP FUNCTION IF EXISTS lastbusinessday(timestamp with time zone, numeric);
+
+CREATE FUNCTION lastbusinessday(p_date timestamp with time zone, p_c_calendar_id numeric) RETURNS timestamp with time zone
```

Comment:
sorry but not calling it `previousBusinessDay` as Cristina suggested?
won't be much more intention revealing and also consistent with `nextBusinessDay`?

Reply:
as the date itself is included i thought it would be a better name